### DISCUSSION TITLE
Codespaces on Enterprise Server roadmap/ETA?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#27426] [Enterprise] [answered]
Greetings,

I'm curios to know if there is a plan for brining code spaces to Enterprise Server, and if so is there any estimate on when we could expect it :)

br,
Granit

**Comment by jungaretti** [comment, marked as answer]
We don't have an ETA to share at this time.

**Reply by kpalf** [reply]
Anything new to share yet?  I see it is now available even on the free tier publicly - so any news yet for GHES in future?
Thanks,
Kev

**Comment by froblesmartin** [comment]
This was also asked at https://github.com/community/community/discussions/5581
